Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM archlinux/base
+
+RUN pacman -Sy --noconfirm arm-none-eabi-binutils \
```

Comment:
How about using `set -e` and maybe `set -x` too?

Reply:
Added